jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -1398,6 +1419,45 @@ private Object envObj(XmlParser.Node node) throws Exception
             return value;
         }
 
+        /**
+         * Check children for all {@code <Property>} and {@code <SystemProperty>} and return
+         * the String representation of any declared {@code default="value"} attributes.
+         *
+         * @param obj the enclosing obj
+         * @param node the XML node
+         * @return a String representing all {@code <Property default="...">} and {@code <SystemProperty default="...">} values appended together
+         */
+        private String defaultValue(Object obj, XmlParser.Node node) throws Exception
+        {
+            StringBuilder ret = new StringBuilder();
+
+            appendDefaultPropertyValues(ret, node);
+
+            return ret.toString();
+        }
+
+        private void appendDefaultPropertyValues(StringBuilder defValues, XmlParser.Node node) throws Exception
+        {
+            for (Object child : node)
+            {
+                if (child instanceof XmlParser.Node)
+                {
+                    XmlParser.Node childNode = (XmlParser.Node)child;
+                    String tag = childNode.getTag();
+                    if ("Property".equals(tag) || "SystemProperty".equals(tag))
+                    {
+                        AttrOrElementNode aoeNode = new AttrOrElementNode(childNode, "Id", "Name", "Deprecated", "Default");
+                        String dftValue = aoeNode.getString("Default");
+                        if (dftValue != null)
+                        {
+                            defValues.append(dftValue);
```

Comment:
@joakim I think this works OK for:
```xml
    <Set><Property name="foo" default="foo"/><Property name="bar" default="bar"/></Set>
```
But will not work for the more complex case where enclosed text is included:
```xml
    <Set>
      <Property name="foo" default="foo"/>
      and
      <Property name="bar" default="bar"/>
    </Set>
```

But then this is only a horrid hack for for 9.4.x and will never EVER be merged to 10.   Right???